### PR TITLE
Normalizing upgrade instructions and reducing need for edits when platforms are added

### PR DESCRIPTION
**Pull request by dwicinas**

### CODE REVIEW

**Comment by dwicinas** [comment]
@drothery-edb Ready for review. There's a lot of rewriting, so it might be easiest to just look at the Deploy version of the upgrade topics, but there's a conflict so a Deploy version is not currently generating. Maybe we can resolve that conflict tomorrow if you can't figure it out yourself.

**Comment by dwicinas** [comment]
@drothery-edb I made the changes we discussed last week where I could. Not all products follow the same pattern. For the FDWs, I didn't combine upgrade commands into one code block because the command varies by platform. Also, I made an educated guess about the upgrade command for Hadoop (you'll recall the cut and paste error from Mongo). With all those caveats, I'd say this is ready for your review, again.

File: product_docs/docs/hadoop_data_adapter/2/upgrading.mdx [hunk 1]
```diff
@@ -6,63 +6,58 @@ redirects:
...
-```shell
-# Update your edb.repo file to access the current EDB repository
-dnf upgrade edb-repo
+To update your repository configuration file:
 
-# Upgrade to the latest version product 
-dnf upgrade edb-as<xx>-hdfs_fdw
-# where <xx> is the EDB Postgres Advanced Server version number
+```shell
+sudo <package-manager> upgrade edb-repo
```
[the remark this answers is not in the document]
Comment (nidhibhammar) [reply product_docs/docs/hadoop_data_adapter/2/upgrading.mdx:16]: I confirmed from Ajay Pal that there is no need to update the edb-repo. We should be updating the actual product packages only.

Comment (drothery-edb) [inline product_docs/docs/hadoop_data_adapter/2/upgrading.mdx:16]:
```suggestion
sudo <package-manager> 
```

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**